src/translations/indexMapper.js

```diff
@@ -119,19 +162,22 @@ class IndexMapper {
   }
 
   /**
-   * Register map which provide some index mappings.
+   * Register map which provide some index mappings. Type of map determining to which collection it will be added.
    *
    * @param {string} uniqueName Name of the index map. It should be unique.
    * @param {IndexMap} indexMap Registered index map updated on items removal and insertion.
    * @returns {IndexMap}
    */
   registerMap(uniqueName, indexMap) {
-    if (this.skipMapsCollection.get(uniqueName) || this.variousMapsCollection.get(uniqueName)) {
+    if (this.trimmingMapsCollection.get(uniqueName) || this.hidingMapsCollection.get(uniqueName) || this.variousMapsCollection.get(uniqueName)) {
```

Comment:
```suggestion
    if (this.trimmingMapsCollection.has(uniqueName) || this.hidingMapsCollection.has(uniqueName) || this.variousMapsCollection.has(uniqueName)) {
```

Reply:
It's not a simple map :-) I haven't implemented `has` and probably it would be useful only for this conditional. I suppose that it's not needed yet.